apps/api/.env.example

```diff
@@ -1,27 +1,29 @@
 # API configuration
-HOST=
-PORT=
-API_KEY=
+HOST= ;e.g. localhost
+PORT= ;e.g. 3001
+API_KEY= ;any string, e.g. some_api_key
 
 # Postgres configuration
-DATABASE_URL=
+DATABASE_URL= ;match the DB you’ve created, e.g. postgres://postgres:postgres@localhost:5432/algomart
 DATABASE_SCHEMA=public
 
 # Algod configuration
-ALGOD_SERVER=
-ALGOD_PORT=
-ALGOD_TOKEN=
-ALGOD_ENV=testnet
+ALGOD_SERVER= ;e.g. http://localhost
+ALGOD_PORT= ;e.g. 4001
+ALGOD_TOKEN= ;e.g. aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+ALGOD_ENV= ;e.g. testnet
 
-# CMS configuration (CMS_ACCESS_TOKEN should match the Admin User > Admin Options > Token field in the CMS UI)
-CMS_ACCESS_TOKEN=
+# CMS configuration
+
+CMS_ACCESS_TOKEN= ;any string, should match the Admin User > Admin Options > Token field in the CMS UI
 CMS_URL=http://localhost:8055
 
 # Web app configuration
 WEB_URL=http://localhost:3000
 
 # Circle API configuration
-CIRCLE_API_KEY=
+
+CIRCLE_API_KEY= ;create a free circle account to generate a key
 CIRCLE_URL=https://api-sandbox.circle.com
```

Comment:
I guess we are already using dev defaults in some places. That's preferred IMO but that might just be a taste thing.

Reply:
The `CIRCLE_API_KEY` has to be unique per project so we can't default one here, might suggest linking directly to the signup page where a sandbox key can be provisioned freely for development purposes (https://my-sandbox.circle.com/signup)